aloscene/mask.py

```diff
@@ -13,20 +22,113 @@ class Mask(aloscene.tensors.SpatialAugmentedTensor):
     """
 
     @staticmethod
-    def __new__(cls, x, *args, **kwargs):
+    def __new__(cls, x, labels: Union[dict, Labels] = None, *args, **kwargs):
         # Load frame from path
         if isinstance(x, str):
             x = load_mask(x)
-            kwargs["names"] = ("C", "H", "W")
+            kwargs["names"] = ("N", "H", "W")
         tensor = super().__new__(cls, x, *args, **kwargs)
+        tensor.add_label("labels", labels, align_dim=["N"], mergeable=True)
+        tensor.add_property("mask_size", x.shape)
```

Comment:
This line does not seem great to me. If the frame get resized, the mask will be resized and this property might not be updated properly. Also it is still possible to get the shape of the mask at any time using self.shape

Reply:
Done. Property removed.